OpenIPSL/Examples/Tutorial/Example_4/Utilities/saveTotalSMIBModel.mo

```diff
@@ -0,0 +1,7 @@
+within OpenIPSL.Examples.Tutorial.Example_4.Utilities;
+function saveTotalSMIBModel "Save the SMIB package as a total model"
+  extends Modelica.Icons.Function;
+  output Boolean ok "True if succesful";
```

Comment:
```suggestion
  output Boolean ok "= true, if successful";
```
See also `Modelica.UsersGuide.Conventions.ModelicaCode.Format`

Reply:
Done!

Reply:
Well, not quite. You could have simply clicked the "Commit suggestion" button and then you would have been done. But now your submitted a different version including a typo. So really it should say: 
`"= true, if successful"`

Reply:
Fixed!